zerver/views/webhooks/github.py

```diff
@@ -14,17 +14,20 @@
 ZULIP_TEST_REPO_ID = 6893087
 
 def is_test_repository(repository):
+    # type: (Dict[str, Any]) -> bool
```

Comment:
I think you should use [`typing.Mapping`](https://docs.python.org/3/library/typing.html#typing.Mapping) instead of `Dict`.